### ISSUE TITLE
Random monsters?

### ISSUE THREAD

**Issue by Dokman** [Done52/Ancestra_Remake#7]
I think that in a map it has a group of monsters but appears randomly?
It's possible to do?

**Comment by Sparksx** [comment]
Yes, the group of monster appears randomly (composition of group)

**Comment by Dokman** [comment]
yes but it has a lot of level if you are level 1 the level of monsters are 7 or 8

**Comment by Sparksx** [comment]
The level of monster depends on the zone, not on your level.. If we set the monster's level according to the player's level, we would open all zones at lower levels. It's possible but it's not one of the main functions of Dofus.
